go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/golang/mock v1.6.0
 	github.com/gophercloud/gophercloud v0.25.0
 	github.com/gophercloud/utils v0.0.0-20210909165623-d7085207ff6d
+	github.com/mcuadros/go-version v0.0.0-20190830083331-035f6764e8d2
```

Comment:
This project is unmaintained, please use https://github.com/hashicorp/go-version instead.

Reply:
> This project is unmaintained, please use https://github.com/hashicorp/go-version instead.

Hashicorp is MPL 2.0 license. I used that first in subctl PR, but had to change due to failing license check.

Reply:
So it is; how about https://github.com/blang/semver/v4?

Reply:
… or https://github.com/coreos/go-semver/ which is already used elsewhere in Submariner.

Reply:
Ran into this issue with semver:

```
OCP version is 4.11.0-rc.1, submariner OVN requires 4.11.0+
```
Should I use something like `4.11.0-rc` as min supported version instead of `4.11.0`?